### PR TITLE
Add recipe timu-spacegrey-theme

### PR DESCRIPTION
**Pull request by macownersclub**
### Brief summary of what the package does

An Emacs color theme inspired by the Sublime Text Spacegray theme.
Prominent color is orange.
Loads of (3rd party) packages are included in the themening.

### Direct link to the package repository

https://gitlab.com/aimebertrand/timu-spacegrey-theme

### Your association with the package

I am the Author of the Theme/Package

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [ ] I have confirmed some of these without doing them

### CODE REVIEW

File: recipes/timu-spacegrey-theme [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(timu-spacegrey-theme :repo "aimebertrand/timu-spacegrey-theme"
+                      :fetcher gitlab
+                      :files ("README.org" "timu-spacegrey-theme.el" "timu-spacegrey-theme.png"))
```
Comment (riscy) [inline recipes/timu-spacegrey-theme:3]:
```suggestion
(timu-spacegrey-theme :repo "aimebertrand/timu-spacegrey-theme")
```
The default recipe should work here -- we usually don't package READMEs and screenshots.

**Review by riscy — COMMENTED** [review]
This looks good to me, however are you aware of the `doom-spacegrey` theme?  It may be a carbon copy of this.

**Comment by macownersclub** [comment]
It seems as a matter of fact almost the exact the same colors.
Doesn't look the same applied, though.

You guys can decide.

**Comment by riscy** [comment]
If it looks different that's all that counts (at least as far as color themes go).  Thanks, and welcome to MELPA. :)

**Pull request merged**